src/manifest.json

```diff
@@ -28,6 +28,6 @@
 				"<all_urls>"
 			]
 		}
-	],
-	"key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAoDtd1NOLQVtfw2NjCJaQvfs1vlUdDbGHESPcFu20fBHGOBdJkRRgd70v+cDeuMzR3d0IQW18oc/XnTb8syGx49a8ummj0cly2bvIr3xzxZe9VoF4FuTKGVrLt0Ru+sVKUUSoVBL1/tS3l4wqZ1V/cNlGlSeqbEznxCVJD3QEk8Q926ucEV6/xQt7G/JPAIPZg3GRIPUvbWL/djEh9oV8HTqQ5gHbtzRFRLckMNEbMev3xzBxrJLKfg3eGqV012Sf0eqE4aW6xCheB0mSEZohPMEkWrAWknEb5P08/Z6zhWS9W0rCzWIewzAl9qEl8KwMCLzn/NIUBt4rugitTNDjWwIDAQAB"
```

Comment:
Without this how do we know the name for the messages?

Reply:
what message are you referring to?

Reply:
The send message first argument of the plug-in name

Reply:
can you elaborate, like is some code somewhere that needs that key value and where is that code?

Reply:
![image](https://user-images.githubusercontent.com/8322045/128660196-ca72197b-c679-4fac-b1ac-d2c216b21cca.png)

Reply:
https://developer.chrome.com/docs/extensions/mv3/manifest/key/

Looks like we should use the [getUrl](https://developer.chrome.com/docs/extensions/reference/runtime/#method-getURL) method

Reply:
https://developer.chrome.com/docs/extensions/reference/runtime/#property-id

Or this

Reply:
![image](https://user-images.githubusercontent.com/8322045/128778921-31094086-732a-4e98-aeb6-1ffe2cae0c14.png)

We do not need to the write the key messsage.

Reply:
Remove it and try :)